specification/communication/data-plane/CallAutomation/stable/2023-03-06/communicationservicescallautomation.json

```diff
@@ -0,0 +1,2285 @@
+            "$ref": "./examples/CallRecording_PauseRecording.json"
+          }
+        }
+      }
+    },
+    "/calling/recordings/{recordingId}:resume": {
+      "post": {
+        "tags": [
+          "Recording"
+        ],
+        "summary": "Resume recording the call.",
+        "operationId": "CallRecording_ResumeRecording",
+        "parameters": [
+          {
+            "name": "recordingId",
+            "in": "path",
+            "description": "The recording id.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            }
+          },
+          "202": {
+            "description": "Returns the resume call recording response."
+          }
+        },
+        "x-ms-examples": {
+          "CallRecording_ResumeRecording": {
+            "$ref": "./examples/CallRecording_ResumeRecording.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "CreateCallRequest": {
+      "description": "The request payload for creating the call.",
+      "required": [
+        "targets",
+        "callbackUri"
+      ],
+      "type": "object",
+      "properties": {
+        "targets": {
+          "description": "The targets of the call.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CommunicationIdentifierModel"
+          }
+        },
+        "sourceCallerIdNumber": {
+          "$ref": "#/definitions/PhoneNumberIdentifierModel",
+          "description": "The source caller Id, a phone number, that's shown to the PSTN participant being invited.\r\nRequired only when calling a PSTN callee."
+        },
+        "sourceDisplayName": {
+          "description": "Display name of the call if dialing out to a pstn number",
+          "type": "string"
+        },
+        "sourceIdentity": {
+          "$ref": "#/definitions/CommunicationUserIdentifierModel",
+          "description": "The identifier of the source of the call"
+        },
+        "operationContext": {
+          "description": "A customer set value used to track the answering of a call.",
+          "type": "string"
+        },
+        "callbackUri": {
+          "description": "The callback URI.",
+          "type": "string"
+        }
+      }
+    },
+    "CommunicationIdentifierModelKind": {
+      "description": "Type of CommunicationIdentifierModel.",
+      "enum": [
+        "unknown",
+        "communicationUser",
+        "phoneNumber",
+        "microsoftTeamsUser"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "CommunicationIdentifierModelKind",
+        "modelAsString": true
+      }
+    },
+    "CommunicationIdentifierModel": {
+      "type": "object",
```

Comment:
Model (at least important ones) should have a good description.

Reply:
I will update all descriptions accordingly.

Reply:
This was updated with common file reference (as Dominik mentioned below)